src/libsystemd-network/sd-radv.c

```diff
@@ -238,6 +304,13 @@ _public_ int sd_radv_start(sd_radv *ra) {
         (void) sd_event_source_set_description(ra->timeout_event_source,
                                                "radv-timeout");
 
+        r = icmp6_bind_router_advertisement(ra->ifindex);
+        if (r < 0)
+                goto fail;
+
+        ra->fd = r;
+        r = 0;
+
         ra->state = SD_RADV_STATE_ADVERTISING;
 
         log_radv("Started IPv6 Router Advertisement daemon");
```

Comment:
" daemon" might be confusing to users... In other messages you just use "Started/stopped IPv6 Router Advertisement".